.TH std::nested_exception::~nested_exception 3 "Apr 19 2014" "1.0.0" "C++ Standard Libary"
.SH Synopsis
   virtual ~nested_exception() = default;  \fI(since C++11)\fP

   Destroys the nested exception object.
